frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Stage 0, "build-stage", based on Node.js, to build and compile the frontend
-FROM node:20 AS build-stage
+FROM node:lts AS build-stage
```

Comment:
I prefer to have a known working version number, `lts` will change in the future with changes to NodeJS that are now still unknown.